pkg/controllers/user/cis/rbac.go

```diff
@@ -0,0 +1,134 @@
+package cis
```

Comment:
You put this in the `user` package. A user handler is one that writes to the downstream cluster. This one does not. It should be in the `management` package.

Reply:
👍